source/Nevermore/Advanced/ReadTransaction.cs

```diff
@@ -714,7 +767,9 @@ RetryPolicy GetRetryPolicy(RetriableOperation operation)
             if (operationsToRetry == RetriableOperation.None)
                 return RetryPolicy.NoRetry;
 
-            return (operationsToRetry & operation) != 0 ? RetryManager.Instance.GetDefaultSqlCommandRetryPolicy() : RetryPolicy.NoRetry;
+            //return (operationsToRetry & operation) != 0 ? RetryManager.Instance.GetDefaultSqlCommandRetryPolicy() : RetryPolicy.NoRetry;
+
+            return RetryManager.Instance.GetDefaultSqlCommandRetryPolicy();
```

Comment:
We are not rolling back on an operation, instead a whole transaction so I short circuited this  for the purposed of POC